README.md

```diff
@@ -1,24 +1,211 @@
+- **Likes on Posts**
+- **Comments on Posts**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p> -->
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+```
+    ruby >= 3.2.0
+    rails >= 7.0
+    postgres >- 15.3
+```
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```bash
+  git clone https://github.com/sunga12/New-Blog-App.git
+```
+
+You need to setup database for these project
+
+```
+  development = blog_app_development
+  test        = blog_app_test
+  production  = blog_app_production
+```
+
+or you can use your own database and change the `config/database.yml`
+
+```yml
+default: &default
+  adapter: postgresql
+  encoding: unicode
+  pool: 5
+  username: [your_username]
+  password: [your_password]
+
+
+development:
+  <<: *default
+  database: [your_database_for_development]
+  host: localhost
+  port: 5432
+
+test:
+  <<: *default
+  database: [your_database_for_test]
+
+production:
+  <<: *default
+  database: [your_database_for_production]
+```
+
+### Install
+
+Install this project with:
+
+```bash
+  cd New-Blog-App
+  bundle install
+```
+
+it will install the required gemfile for running the project
+
+### Usage
+
+to use this project:
+
+```ruby
+   rails s
+```
+
+it will run the the server on `localhost:3000`
+
+<!-- ### Test
+
+to run test in these this project:
+
+```ruby
+   rspec
+```
+
+it will run the all the unit test of these project
+
+ -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Author <a name="author"></a>
+
+👤 **Sungabanja Thawethe**
+
+- GitHub: [@Sunga12](https://github.com/sunga12)
+- LinkedIn: [Sungabanja Thawethe](https://linkedin.com/in/sungabanja-thawethe)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- **Add UI System**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project you can share this project to your friend
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank microverse for this project
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] Dear @sunga12, Great you have added a readme file, But currently your MIT linsense file link is broken. For a professional readme documentation, we need to have a working link all the way. Please address this.